### DISCUSSION TITLE
Improvement option for EU compatibility 2.4 GHz

### DISCUSSION THREAD

**Discussion by franz4949** [ExpressLRS/ExpressLRS#1749] [Ideas]
EU compatibility for the 2.4 GHz system requires to follow the EU ETSI EN 300 328 regulation
 => https://www.etsi.org/deliver/etsi_en/300300_300399/300328/02.02.02_60/en_300328v020202p.pdf 
ELRS v3 with its actual Release Candidate 1 does follow that regulation by applying the adaptive mode operation, known as LBT, with max transmit power of 100 mW.

LBT operation does check whether next channel is free for transmission and when found to be already occupied it is not allowed to transmit, it then will look for next channel in the hopping table. That channel may however also be occupied already, so it has to wait again with sending. In the past there have been discussions whether the system might be allowed to switch from adaptive operation mode to non-adaptive operation mode when next channel is detected to be occupied. I have now contacted the German TUV Sud and talked with a person working in the TUV Sud. His job is testing wireless hardware in the TUV Sud test lab and among others he is also familiar with ETSI EN 300 328. He confirmed on the phone that the code self-evidently does allow the equipment to be operated in adaptive and non-adaptive mode and that switching between that modes is allowed. And furthermore he mentioned that I did correctly and exact quote the clauses on which this is based. The relevant code sections are:

ETSI section 4.2.2 Adaptive and non-adaptive equipment
Adaptive equipment may have more than one adaptive mode implemented. Adaptive equipment is allowed to operate in
a non-adaptive mode. Equipment is allowed to switch between any of these modes.
Unless otherwise specified, the equipment shall comply with the corresponding requirements in each of the modes in
which it can operate.
The manufacturer shall declare whether the equipment is adaptive equipment or non-adaptive equipment. In case of
adaptive equipment, the manufacturer shall declare all adaptive modes in addition to whether the equipment can also
operate in a non-adaptive mode. See also clause 5.4.1.

and ETSI section 5.4.1 Product Information
The following information shall be stated by the supplier in order to carry out the test suites and/or to declare
compliance to technical requirements. This information shall be included in the test report. The application form in annex E can be used for that:
c) with regard to adaptivity, the type of equipment: non-adaptive equipment, adaptive equipment or equipment
that can operate in both an adaptive and non-adaptive mode

And here's the web page of that TUV Sud section => https://www.tuvsud.com/de-de/dienstleistungen/produktpruefung-und-produktzertifizierung/funkpruefung

So it should now be sufficiently clear that the transmitting system is allowed to switch between adaptive (LBT) mode operation with a max. RF output power of 100 mW and non-adaptive mode operation with a RF output power to be determined on basis of the calculation formulas given in the respective code section 4.3.1.2 of ETSI EN 300 328.

Based on aforementioned the actual performance of the EU compatible 2.4 GHz system might be improved by introducing such operation mode switching from adaptive (LBT) mode to non-adaptive operation mode (and back again) and transmit with reduced RF power instead of not sending in case of identified occupied channel.

**Comment by JyeSmith** [comment]
Looks like this might be a goer!

Would you like to look at a future PR to implement the changes?  If so please checkout this PR for some background info.  We need to be a little careful when power levels are actually changed.

https://github.com/ExpressLRS/ExpressLRS/pull/1734

**Reply by franz4949** [reply]
Thank you
